src/Middleware/RequestThrottling/src/RequestThrottlingMiddleware.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.RequestThrottling;
+using Microsoft.AspNetCore.RequestThrottling.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Aspnetcore.RequestThrottling
```

Comment:
```suggestion
namespace Microsoft.AspNetCore.RequestThrottling
```

Reply:
This one is a little more than wordsmithing. It's actually putting this class in a completely different namespace because namespaces are case-sensitive.